src/ml-service/packages/sensor_location/locate/modules/locate_helper.py

```diff
@@ -0,0 +1,193 @@
+import pandas as pd
+import numpy as np
+from sklearn.preprocessing import StandardScaler
```

Comment:
When we start refactoring our codebase, this file could be moved to: `AirQo-api/src/locate/helpers/locate.py` The file could also be renamed to something that truly reflects the content of the helper file.